src/main/java/salesforce/config/Param.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package salesforce.config;
+
+public class Param {
```

Comment:
The class name doesn't provide enough information, could probably be set toCredentialParams